packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -109,7 +109,7 @@ describe('autocomplete-js', () => {
             </div>
           </form>
           <div
-            class="aa-Dropdown"
+            class="aa-ListContainer"
```

Comment:
Any better alternative to this?

Reply:
I don't think dropdown is really a bad name

Reply:
It's going to have `ul`, right? Then.. I guess ListContainer is good.

Reply:
I think something with "results" could also work

Reply:
@Haroenv "dropdown" loses it sense in the context of modal searches like DocSearch v3, so I'm unsure.

Reply:
panel maybe?

Reply:
ListsContainer or ResultsContainer seems best to me. Maybe also something like Suggestions, Collections or Sources, since you renamed that?

Reply:
It's important to note that this term will also replace the main `getDropdownProps` API:

https://github.com/algolia/autocomplete.js/blob/a916aea48743eaa3e97e1e421aa1ac6986fa0e83/packages/autocomplete-core/src/createAutocomplete.ts#L81

Perhaps "Panel" makes sense?

Reply:
Panel came back several times, Menu isn't a bad name neither

Reply:
I renamed "ListContainer" to "Panel" in a600ed256dc6a1750b3d2516e5e473306dea296c. That's good enough for now.